wandb/sdk/wandb_lite_run.py

```diff
@@ -0,0 +1,241 @@
+import datetime
+import logging
+import platform
+import typing
+
+from wandb import __version__, util
+from wandb import errors as wandb_errors
+from wandb.apis import public
+
+from .artifacts import artifact_saver
+from .internal import file_stream
+from .internal.file_pusher import FilePusher
+from .internal.internal_api import Api as InternalApi
+from .internal.thread_local_settings import _thread_local_api_settings
+from .lib import config_util, runid
+from .lib.ipython import _get_python_type
+
+if typing.TYPE_CHECKING:
+    from wandb.sdk.artifacts.artifact import Artifact
+
+logger = logging.getLogger(__name__)
+
+_p_api = None
+
+
+def wandb_public_api() -> public.Api:
+    global _p_api
+    if _p_api is None:
+        _p_api = public.Api(timeout=30)
+    return _p_api
+
+
+def assert_wandb_authenticated() -> None:
+    authenticated = (
+        _thread_local_api_settings.cookies is not None
+        or wandb_public_api().api_key is not None
+    )
+    if not authenticated:
+        raise wandb_errors.AuthenticationError(
+            "Unable to log data to W&B. Please authenticate by setting WANDB_API_KEY or running `wandb init`."
+        )
+
+
+# We disable urllib warnings because they are noisy and not actionable.
+logging.getLogger("urllib3").setLevel(logging.ERROR)
+logging.getLogger("urllib3.connectionpool").setLevel(logging.ERROR)
+
+WANDB_HIDDEN_JOB_TYPE = "sweep-controller"
```

Comment:
curious why is this the job_type?

Reply:
The backend has a bunch of existing logic to hide `sweep-controller` jobs from the UI and tracked hour calculations.  It's super hacky but does the trick 🙃